docs/02.API-REFERENCE.md

```diff
@@ -3868,6 +3868,114 @@ example (void)
 - [jerry_create_promise](#jerry_create_promise)
 - [jerry_promise_state_t](#jerry_promise_state_t)
 
+## jerry_from_property_descriptor
+
+**Summary**
+
+This API function is equivalent to FromPropertyDescriptor operation.
+Based on ECMA 262 v11 6.2.5.4 this is in the form of `FromPropertyDescriptor`.
+
+*Notes*:
+- Returned value must be freed with [jerry_release_value](#jerry_release_value) when it
+  is no longer needed.
+
+**Prototype**
+
+```c
+jerry_value_t
+jerry_from_property_descriptor (const jerry_property_descriptor_t *src_prop_desc_p)
+```
+
+- `src_prop_desc_p` - the input property descriptor.
+- return
+  - [jerry_value_t](#jerry_value_t)
+  - jerry value - if success
+  - value marked with error flag - otherwise
+
+**Example**
+
+[doctest]: # (test="compile")
+
+```c
+#include <jerryscript.h>
+
+static void
+example (void)
+{
+  jerry_value_t object = jerry_create_object ();
+  jerry_value_t prop_name = jerry_create_string_from_utf8 ((jerry_char_t *) "value");
+  jerry_value_t value = jerry_create_boolean (true);
+
+  jerry_set_property (object, prop_name, prop_name);
+
+  jerry_property_descriptor_t prop_desc;
+  jerry_init_property_descriptor_fields (&prop_desc);
+
+  jerry_get_own_property_descriptor (object, prop_name, &prop_desc);
+
+  jerry_value_t from_object = jerry_from_property_descriptor (&prop_desc);
+
+
+  jerry_release_value (object);
+  jerry_release_value (prop_name);
+  jerry_release_value (value);
+  jerry_release_value (from_object);
+  jerry_free_property_descriptor_fields (&prop_desc);
+}
+```
+
+## jerry_to_property_descriptor
+
+**Summary**
+
+This API function is equivalent to ToPropertyDescriptor operation.
+Based on ECMA 262 v11 6.2.5.5 this is in the form of `ToPropertyDescriptor`.
+
+*Notes*:
+- Returned value must be freed with [jerry_release_value](#jerry_release_value) when it
```

Comment:
It returns with a bool, so this is invalid. Should it return with an error when the conversion is unsuccessful?